.travis.yml

```diff
@@ -1,19 +1,16 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
-  - linux
-  - osx
+    - linux
```

Comment:
I copied the entire contents from a travis file in another package which probably lead to this :sweat_smile: I'll change it :smile: